src/main/java/org/wirvsvirus/locoronando/category/Category.java

```diff
@@ -0,0 +1,42 @@
+package org.wirvsvirus.locoronando.category;
+
+import java.util.UUID;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Category {
+  @Id
+  private UUID uniqueId;
+  private String name;
+  private String description;
+  private int handlerId;
+
+  public UUID uniqueId() {
```

Comment:
@christianor used Lombok to do that. Should we use it globally?

Reply:
I wasn't paying attention, I wouldn't want to use Lombok.